src/main/java/de/bonndan/nivio/ProcessingErrorEvent.java

```diff
@@ -20,9 +26,16 @@ public ProcessingException getException() {
         return exception;
     }
 
-    @Override
-    public Landscape getLandscape() {
-        return exception.getLandscape();
+    @JsonSerialize(using = PLS.class)
+    public LandscapeDescription getLandscape() {
```

Comment:
It is currently implemented like that, but KPIFactory operates on Landscape and throws a ProcessingException, but does not know LandscapeDescription, so the exception and the resulting event do not have anything.

The proper solution here were to run the KPI "init" stuff earlier (this also makes sense, since it is about parsing the config) when LandscapeDescription is still available. I think we should open a new ticket for that.

Reply:
@bonndan: Agreed, can you open the ticket since you can probably describe the underlying mechanisms better than me in this case? :)